cupyx/scipy/ndimage/_interp_kernels.py

```diff
@@ -302,7 +302,7 @@ def _generate_interp_custom(coord_func, ndim, large_int, yshape, mode, cval,
                 dcoord = c_{j};''')
             else:
                 ops.append(f'''
-                {int_t} cf_{j} = ({int_t})lrint((double)c_{j});''')
+                {int_t} cf_{j} = ({int_t})floor((double)c_{j} + 0.5);''')
```

Comment:
Maybe we can get a (tiny bit) more of performance here by doing `__dadd_rd((double) c_{j}, 0.5)`


https://docs.nvidia.com/cuda/cuda-math-api/group__CUDA__MATH__INTRINSIC__DOUBLE.html#group__CUDA__MATH__INTRINSIC__DOUBLE_1gd39098ab6fd26648cffe5db8e2a45158